README.md

```diff
@@ -16,20 +18,20 @@ Enter "Access Key ID" and "Secret Access Key" provided by an AWS administrator
 If you're using macOS go to Macintosh HD > Applications > Python folder > double click on "Install Certificates.command" file.
 
 * Configure  
-`export S3_BUCKET_NAME=bucket-name`
+`export S3_BUCKET_NAME=bucket-name`  
```

Comment:
extra spaces

Reply:
They're for Markdown formatting, forcing a newline.